test/useCases/signInUser.integration.test.ts

```diff
@@ -49,8 +49,11 @@ describe("SigninUser", () => {
       groupId: selectedGroup.id,
       visibleForces: "001,004,",
       visibleCourts: "B01,B41ME00",
+      exclusionList: [],
+      inclusionList: [],
```

Comment:
you might want to remove exclusionList and inclusionList. Those will be removed at one point. They have been replaced in functionality by visibleForces, visibleCourts and excludedTriggers

Reply:
Review comment 7